content/en/getting_started/synthetics/_index.md

```diff
@@ -5,51 +5,53 @@ further_reading:
     - link: 'https://learn.datadoghq.com/course/view.php?id=39'
       tag: 'Learning Center'
       text: 'Introduction to Synthetic Tests'
-    - link: '/synthetics/browser_tests'
-      tag: 'Documentation'
-      text: 'Learn more about browser tests'
     - link: '/synthetics/api_tests'
       tag: 'Documentation'
       text: 'Learn more about API tests'
-    - link: '/synthetics/browser_tests/#subtests'
+    - link: '/synthetics/multistep'
       tag: 'Documentation'
-      text: 'Create a browser subtest'
-    - link: '/synthetics/settings/'
+      text: 'Learn more about Multistep API tests'
+    - link: '/synthetics/browser_tests'
       tag: 'Documentation'
-      text: 'Configure advanced Synthetic Monitoring settings'
+      text: 'Learn more about Browser tests'
+    - link: '/synthetics/private_locations'
+      tag: 'Documentation'
+      text: 'Learn more about Private locations'
       
 ---
 
 ## Overview
 
-Datadog Synthetic Monitoring allows you to monitor uptime, identify regional issues, and ensure key business transactions can perform on your applications.
+Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. Datadog tracks the performance of your webpages and APIs from the backend to the frontend, and at various network levels (`HTTP`, `TCP`, `SSL`, `DNS`, and `ICMP`) in a controlled and stable way, alerting you in case of faulty behavior (regression, broken feature, high response time, unexpected status code, etc.).
```

Comment:
```suggestion
Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. Datadog tracks the performance of your webpages and APIs from the backend to the frontend, and at various network levels (`HTTP`, `TCP`, `SSL`, `DNS`, and `ICMP`) in a controlled and stable way, alerting you in case of faulty behavior (such as regression, broken feature, high response time, or unexpected status code).
```